mewarpx/tests/test_capacitive_discharge.py

```diff
@@ -0,0 +1,87 @@
+"""Test full 1D diode run with diagnostics."""
+import pytest
+import numpy as np
+
+from mewarpx import mwxutil
+
+
+@pytest.mark.parametrize(
+    ("name"),
+    [
+        'Run2D',
+        'Run2D_RZ',
+        'Run3D'
+    ]
+)
+def test_run1D_alldiags(capsys, name):
+    basename = "Run"
+    use_rz = 'RZ' in name
+    dim = int(name.replace(basename, '')[0])
+
+    # Initialize and import only when we know dimension
+    mwxutil.init_libwarpx(ndim=dim, rz=use_rz)
+    from mewarpx import testing_util
+    from mewarpx.setups_store import diode_setup
+
+    # Include a random run number to allow parallel runs to not collide. Using
+    # python randint prevents collisions due to numpy rseed below
+    testing_util.initialize_testingdir(name)
+
+    # Initialize each run with consistent, randomly-chosen, rseed. Use a random
+    # seed instead for initial dataframe generation.
+    # np.random.seed()
+    np.random.seed(92160881)
+
+    # Specific numbers match older run for consistency
+    FREQ = 13.56e6  # MHz
+    DT = 1.0 / (400 * FREQ)
+    DIAG_STEPS = 2
+    DIAG_INTERVAL = DIAG_STEPS*DT
+    VOLTAGE = 450.0
+    run = diode_setup.DiodeRun_V1(
+        dim=dim,
+        rz=use_rz,
+        V_ANODE_CATHODE=VOLTAGE,
+        V_ANODE_EXPRESSION="%.1f*sin(2*pi*%.5e*t)" % (VOLTAGE, FREQ),
+        D_CA=0.067,
+        INERT_GAS_TYPE='He',
+        N_INERT=9.64e20,  # m^-3
+        T_INERT=300.0,  # K
```

Comment:
I believe that only P_INERT, and T_INERT will be passed from the PICMI script, and we will have to calculate N_INERT from those using the ideal gas law. I've implemented the ideal gas law function in `mewarpx/mewarpx/util.py` on the `kzhu/feature_mcc-wrapper` branch. The function is used in `mewarpx/mewarpx/mcc_wrapper.py`

Reply:
Yeah I left a comment on your PR suggesting we add N_INERT as an input option. So I'll throw this one back on you ;-)